src/expire.c

```diff
@@ -161,16 +174,112 @@ static inline int isExpiryTableValidForSamplingCb(hashtable *ht) {
     return C_OK;
 }
 
-void activeExpireCycle(int type) {
+static inline int activeExpireFieldsCheckTimeLimitReached(
```

Comment:
lets put some documentation about this function checks and guarantees. what each of the parameters is expected to  provide etc...